databricks/koalas/series.py

```diff
@@ -635,7 +635,7 @@ def rfloordiv(self, other) -> "Series":
     koalas = CachedAccessor("koalas", KoalasSeriesMethods)
 
     # Comparison Operators
-    def eq(self, other) -> bool:
+    def eq(self, other) -> "Series":
```

Comment:
Good catch! Thanks!

Reply:
I'm wondering if we want to add a docstring example here for list-like Python objects.

Reply:
Added, thanks :)